src/components/ItemFilters/ItemFilters.tsx

```diff
@@ -37,6 +41,9 @@ const ItemFilters = ({
   appliedFilters = { location: [], format: [], status: [], year: [] },
   filtersAreApplied = false,
 }: ItemFilterContainerProps) => {
+  // We have to set the year value in state to be able to test form control in jest.
```

Comment:
The problem was occurring because we were using event.target.value to grab the value entered into the year form, so the integration test for submitting the form and updating the router ("updates the router when year filter is submitted") was failing.